$OpenBSD: patch-src_pty_c,v 1.8 2010/10/13 16:20:23 ajacoutot Exp $

Party revert to previous behaviour of returning -1, instead of failing
to compile.

--- src/pty.c.orig	Sat Oct  9 05:41:51 2010
+++ src/pty.c	Wed Oct 13 18:10:10 2010
@@ -795,7 +795,8 @@ _vte_pty_ptsname(int master,
                     "%s failed: %s", "ioctl(TIOCGPTN)", g_strerror(errno));
         return NULL;
 #else
-#error no ptsname implementation for this platform
+#warning no ptsname implementation for this platform
+	return NULL;
 #endif
 }
 
@@ -903,7 +904,8 @@ _vte_pty_unlockpt(int fd,
         }
         return TRUE;
 #else
-#error no unlockpt implementation for this platform
+#warning no unlockpt implementation for this platform
+	return -1;
 #endif
 }
 
